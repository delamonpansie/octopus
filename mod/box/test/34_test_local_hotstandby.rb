#!/usr/bin/ruby

$: << File.dirname($0) + '/lib'
require 'run_env'

class Env < RunEnv
  def config
    super + <<EOD
local_hot_standby = 1
EOD
  end
end

Env.connect_eval do
  insert ['0', 'a', 'b', 'c', 'd']
  select ['0']
end

