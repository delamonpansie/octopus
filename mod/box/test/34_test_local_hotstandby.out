# box.insert(["0", "a", "b", "c", "d"])
1

# box.select(["0"])
[["0", "a", "b", "c", "d"]]

